src/breadcrumb/breadcrumb.component.ts

```diff
@@ -71,22 +71,34 @@ const MINIMUM_OVERFLOW_THRESHOLD = 4;
 				</ng-template>
 			</cds-breadcrumb-item>
 			<cds-breadcrumb-item>
-				<cds-overflow-menu>
-					<li class="cds--overflow-menu-options__option"
-						*ngFor="let item of overflowItems">
-						<a class="cds--overflow-menu-options__btn"
-							href="{{item?.href}}"
-							(click)="navigate($event, item)"
-							style="text-decoration: none;">
-							<ng-container *ngIf="!item?.template">{{item?.content}}</ng-container>
-							<ng-template
-								*ngIf="item?.template"
-								[ngTemplateOutlet]="item?.template"
-								[ngTemplateOutletContext]="{ $implicit: item }">
-							</ng-template>
-						</a>
-					</li>
-				</cds-overflow-menu>
+				<cds-icon-button
+					[buttonNgClass]="{ 'cds--overflow-menu': true }"
+					type="button"
+					kind="ghost"
+					size="md"
+					[description]="overflowMenuDescription">
+					<ng-template #overflowMenuTrigger>
+						<svg class="cds--overflow-menu__icon" cdsIcon="overflow-menu--horizontal" size="16"></svg>
+					</ng-template>
+					<cds-overflow-menu
+						[customTrigger]="overflowMenuTrigger"
+						triggerClass="cds--btn--icon-only">
+						<li class="cds--overflow-menu-options__option"
+							*ngFor="let item of overflowItems">
+							<a class="cds--overflow-menu-options__btn"
+								href="{{item?.href}}"
+								(click)="navigate($event, item)"
+								style="text-decoration: none;">
+								<ng-container *ngIf="!item?.template">{{item?.content}}</ng-container>
+								<ng-template
+									*ngIf="item?.template"
+									[ngTemplateOutlet]="item?.template"
+									[ngTemplateOutletContext]="{ $implicit: item }">
+								</ng-template>
+							</a>
+						</li>
+					</cds-overflow-menu>
+				</cds-icon-button>
```

Comment:
Functionality works, but this fails accessibility. This is because we have a button within a button. Still thinking of how we can fix this... may need a custom solution or need to extend overflow menu perhaps where we project content for the overflow menu trigger.

I'll look into this and get back to you on how we can approach this.

Reply:
I'm going to make adjustments to the PR. It seems like OverflowMenu now supports tooltip, so we take the same approach as icon-button for it.